File: 프로그래머스/lv1/12925. 문자열을 정수로 바꾸기/문자열을 정수로 바꾸기.cs
public class Solution {
    public int solution(string s) {
        bool isMinus = false;
        if(string.Equals(s[0], "-")){
            isMinus = true;
        }
        int answer = int.Parse(s);
        if (isMinus)
        {
            answer *= -1;
        }
        return answer;
    }
}

File: 프로그래머스/1/12925. 문자열을 정수로 바꾸기/문자열을 정수로 바꾸기.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

int solution(string s) {
    int answer = 0;
    bool isMimus = false;
    
    if(s[0] == '-')
    {
        answer = stoi(s.substr(1,sizeof(s)-1));
}
    answer = stoi(s);
    return answer;
}